Update BlueJeans link for PCM and add calendar link (#422)
containers/podman.io: docs/installation.md

If any dependencies cannot be installed or are not sufficiently current, they have to be built from source.
This will mainly affect Debian, Ubuntu, and related distributions, or RHEL where no subscription is active (e.g. Cloud VMs).

> #### Golang

Reviewer: ```suggestion
#### Go
```

> Be careful to double-check that the version of Golang is new enough (i.e., `go version`). As of May 2024, version 1.21.x or higher is required.

Reviewer: ```suggestion
Be careful to double-check that the version of Go is new enough (i.e., `go version`). As of May 2024, version 1.21.x or higher is required.
```

The current minimum required version can always be found in the [go.mod](https://github.com/containers/podman/blob/main/go.mod) file.

> If needed, Golang kits are available at https://golang.org/dl/. Alternatively, go can be built from source as follows

Reviewer: ```suggestion
If needed, Go binary and source releases are available at https://golang.org/dl/. Alternatively, Go can be built from source as follows
```

### Get Source Code

> First, ensure that the `go version` that is found first on the $PATH is 1.21.x or higher. Instruction [above](#golang) will help you compile newer version of Go if needed. Then we can build Podman:

Reviewer: I think (#golang) would also have to change here based on how you handle line 446.
